components/tooltip/tooltip.js

```diff
@@ -859,6 +862,11 @@ class Tooltip extends RtlMixin(LitElement) {
 		}, 500);
 	}
 
+	_onTooltipShowOther() {
+		// only allow one tooltip showing at a time
+		this.hide();
```

Comment:
It would complicate things a lot, but one scenario this doesn't handle is your first screenshot in the PR description: you have a tooltip that shows on-focus or on-hover, you focus on it, the tooltip shows, then you hover over another element, the tooltip hides, then you stop hovering... despite focus still being on the first element, the tooltip won't come back.

It's a simplified solution, but we could: also listen for `d2l-tooltip-hide` and if we were hidden show ourselves again. That tracking around "if we were hidden" would have to reset when the tooltip was shown/hidden through normal means. It would also only work for one tooltip -- if multiple were hidden (that'd be weird right?) then only one could actually show itself again.

Anyway, not a big deal at all -- not sure if the complexity of the solutions is worth it.

Reply:
I see, yeah, currently they would have to hover over it again, or move focus away and then back again as if they had used `Escape` to dismiss it. Would it be weird to show the original tooltip again after moving the mouse away from the second target - the user would think it's been dismissed, but then it reappears?

Reply:
Yeah it might actually make it look even more buggy and confusing eh. Meh, good with whatever.